proto/data.proto

```diff
@@ -316,6 +316,19 @@ message TimeSeriesData {
   repeated TimeSeriesDatapoint datapoints = 3;
 }
 
+// TimeSeriesQueryResponse is the response to a query over time series data. It
+// is similar to the TimeSeriesData message, but contains a list of sources
+// instead of a single source.
+message TimeSeriesQueryResponse {
```

Comment:
I'm hesitant to use the `Response` suffix here given that it's otherwise the exclusive domain of RPC response messages. How about `TimeSeriesQueryResult` as an alternate?